src/config.c

```diff
@@ -458,6 +458,15 @@ int config_parse_conf(struct config *cfg, const struct conf *conf)
 			    &cfg->net.use_getaddrinfo);
 	(void)conf_get_str(conf, "net_interface",
 			   cfg->net.ifname, sizeof(cfg->net.ifname));
+	if (0 == conf_get(conf, "net_af", &pl)) {
+		if (0 == pl_strcasecmp(&pl, "4"))
```

Comment:
Maybe `all` or `any` for default `AF_UNSPEC` would be nice too.

Reply:
Thanks for the comment.  If "any", then may be the other values would need to be "ipv4" and "ipv6", instead of "4" or "6".

BUT, net_interface does not have "default" value either.  In my app, there will be "any" option, but I don't think it is necessary in config.

Reply:
The default value will be explained in wiki/Configuration.

Reply:
From my point of view "ipv4" and "ipv6" are more self-explaining compared to "4" and "6".

Reply:
Done.